dts/bindings/dma/st,stm32-dma-v2.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2019, Song Qiang <songqiang1304521@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32 DMA controller (V2)
+
+  The STM32 DMA is a general-purpose direct memory access controller
+  capable of supporting 5 or 6 or 7 or 8 independent DMA channels.
+  Each channel can have up to 8 requests.
+  DMA clients connected to the STM32 DMA controller must use the format
+  described in the dma.txt file, using a four-cell specifier for each
+  channel: a phandle to the DMA controller plus the following four integer cells:
+    1. channel: the dma stream from 1 to <dma-requests>
+    2. slot: DMA periph request ID
+    3. channel-config: A 32bit mask specifying the DMA channel configuration
+    which is device dependent:
+        -bit 6-7 : Direction  (see dma.h)
+       	       0x0: MEM to MEM
+       	       0x1: MEM to PERIPH
+       	       0x2: PERIPH to MEM
+       	       0x3: reserved for PERIPH to PERIPH
+        -bit 9 : Peripheral Increment Address
+       	       0x0: no address increment between transfers
+       	       0x1: increment address between transfers
+        -bit 10 : Memory Increment Address
+       	       0x0: no address increment between transfers
+       	       0x1: increment address between transfers
+        -bit 11-12 : Peripheral data size
+       	       0x0: Byte (8 bits)
+       	       0x1: Half-word (16 bits)
+       	       0x2: Word (32 bits)
+       	       0x3: reserved
+        -bit 13-14 : Memory data size
+       	       0x0: Byte (8 bits)
+       	       0x1: Half-word (16 bits)
+       	       0x2: Word (32 bits)
+       	       0x3: reserved
+        -bit 15: Reserved
+        -bit 16-17 : Priority level
+       	       0x0: low
+       	       0x1: medium
+       	       0x2: high
+       	       0x3: very high
+    4. features: A 32bit bitfield value specifying DMA features
+        -bit 0-1: DMA FIFO threshold selection (not USED for DMA V2)
```

Comment:
@FRASTM, I'm not sure about this part.
Does it mean that "features" has no use for V2 ?
(Even if not used, it should be kept for to simplify things at users level, but then I need to make it clear).

Reply:
yes fifo threshold is not used in case of DMA V2

Reply:
ok, txs. I've made it clear in the binding.